charts/aws-calico/README.md

```diff
@@ -0,0 +1,66 @@
+# Calico on AWS
+
+This chart installs Calico on AWS: https://docs.aws.amazon.com/eks/latest/userguide/calico.html
+
+## Prerequisites
+
+- Kubernetes 1.11+ running on AWS
+
+## Installing the Chart
+
+First add the EKS repository to Helm:
+
+```shell
+helm repo add eks https://aws.github.io/eks-charts
+```
+
+Install the Calico CRDs:
+
+```shell
+kubectl apply -k github.com/aws/eks-charts/tree/master/stable/aws-calico/crds
```

Comment:
Can these not be installed w/ helm?  If not, can these be uploaded to the release assets in the aws-calico repo? I don't think it's a good practice to download crds directly from the default branch.

Reply:
sure Brandon, I have exported these from eks-charts. Will fix this as part of following PR.